indexer/feature.hpp

```diff
@@ -36,7 +37,6 @@ class EditableMapObject;
 class FeatureBase
 {
 public:
-
   using TBuffer = char const *;
```

Comment:
s/TBuffer/Buffer/

Reply:
I think I'll revert as many indexer/feature.hpp as possible because we currently have another PR that rewrites it.